### ISSUE TITLE
Decide on board structure: Do shops manage thing on their own boards?

### ISSUE THREAD

**Issue by kwurst** [LibreFoodPantry/Community#52]
Not sure if we have consistency between Workflow Story Map and the latest thinking on boards as in https://github.com/LibreFoodPantry/Community/issues/37#issuecomment-510214380

**Comment by kwurst** [comment]
@StoneyJackson

**Comment by StoneyJackson** [comment]
I wrote an initial structure for boards.

* Vaguely referred to in https://github.com/LibreFoodPantry/ProjectTemplate/blob/master/CONTRIBUTING.adoc
* Specifically discussed here: https://github.com/LibreFoodPantry/ProjectTemplate/blob/master/docs/dev/issue-tracker.adoc

**Comment by StoneyJackson** [comment]
I just reread this. And I now have a different understanding of the question. So let me try again.

Ideally we want shop members to help groom issues. They should be able to move items on their shop and team boards. That's why we were talking about giving them reporter privileges on GitLab.

Is this a better answer?
